src/panel/poi/PoiBlockContainer.jsx

```diff
@@ -27,7 +27,7 @@ export default class PoiBlockContainer extends React.Component {
     const displayCovidInfo = this.props.covid19Enabled && blocks.find(b => b.type === 'covid19');
 
     return <div className="poi_panel__info">
-      {displayCovidInfo && <CovidBlock block={covidBlock} />}
+      {displayCovidInfo && <CovidBlock block={covidBlock} poi={this.props.poi} />}
```

Comment:
Maybe we can pass only the country code as props, so the `CovidBlock` doesn't need to know how to access this info from a full POI object.